src/utils/helpers/poller/poller.js

```diff
@@ -126,7 +127,10 @@ function getFunctions(functions) {
     handleError: functions.handleError || null,
     conditionMet: functions.conditionMet || (() => { return false; }),
     conditionNotMetCallback: functions.conditionNotMetCallback || (() => { return false; }),
-    terminate: functions.terminate || (() => { return false; })
+    terminate: functions.terminate || (() => { return false; }),
+    onMaxRetries: functions.onMaxRetries || (() => {
+      console.warn('The poller has made too many requests - terminating poll'); // eslint-disable-line no-console
```

Comment:
This should probably use the logger rather than warning itself.

This will stop anything showing in production and also concentrates the no-console ESLint ignore to one place